packages/tools/webpack-fluid-loader/package.json

```diff
@@ -62,6 +62,7 @@
 		"temp-directory": "nyc/.nyc_output"
 	},
 	"dependencies": {
+		"@fluid-internal/client-utils": "workspace:~",
```

Comment:
I think the server dep can also be removed in this package?

Reply:
`@fluidframework/server-services-client": "^1.0.1` from the `@fluid-tools/webpack-fluid-loader`.